assist/ephem.py

```diff
@@ -39,39 +41,45 @@ class Ephem(Structure):
     Main object used for all ASSIST Ephemeris operations.
     Not tied to a particular REBOUND simulation.
     """
-    
-    def __init__(self, planets_path=None, asteroids_path=None):
+
+    def __init__(self, planets_path: Optional[str] = None, asteroids_path: Optional[str] = None):
         if planets_path is not None:
-            planets_path = planets_path.encode("ascii")
```

Comment:
Here, by the way, is another thing mypy didn't like. I think the code is clearly more readable before this change, but mypy disagrees.